blockchain/api.go

```diff
@@ -1369,7 +1518,195 @@ func (api *BasicSimpleGatekeeper) Payout(ctx context.Context, key *ecdsa.Private
 	return api.contract.Payout(opts, to, value, txNumber)
 }
 
+func (api *BasicSimpleGatekeeper) PackPayoutTransactionData(to common.Address, value *big.Int, txNumber *big.Int) ([]byte, error) {
+	gateABI, err := abi.JSON(bytes.NewBufferString(marketAPI.SimpleGatekeeperABI))
+	if err != nil {
+		return nil, err
+	}
+
+	return gateABI.Pack("Payout", to, value, txNumber)
+}
+
+func (api *BasicSimpleGatekeeper) UnpackPayoutTransactionData(data []byte) (*SimpleGateTransaction, error) {
+	if len(data) == 0 {
+		return nil, fmt.Errorf("data is empty")
+	}
+
+	if len(data) < 100 {
```

Comment:
You're requiring len=100 but below using only 96 bytes. WTF?

Reply:
4 byte for method definition
3x32 byte for parameters

Reply:
oh, you're cutting the header, got it.